### ISSUE TITLE
SQL layer new sqlfmt configuration will cause sqlup configuration error

### ISSUE THREAD

**Issue by rgb-24bit** [syl20bnr/spacemacs#12147]
#### Description :octocat:
These two configurations seem to have a conflict:

```elisp
(defun sql/init-sqlfmt ()
  (use-package sqlfmt
    :commands sqlfmt-buffer
    :init
    (spacemacs/set-leader-keys-for-major-mode 'sql-mode
      "=" 'sqlfmt-buffer)))

(defun sql/init-sqlup-mode ()
  (use-package sqlup-mode
    :defer t
    :init
    (progn
      (add-hook 'sql-mode-hook 'sqlup-mode)
      (unless sql-capitalize-keywords-disable-interactive
        (add-hook 'sql-interactive-mode-hook 'sqlup-mode))
      (spacemacs/set-leader-keys-for-major-mode 'sql-mode
        ;; Sqlup requires = as a prefix key
        "=c" 'sqlup-capitalize-keywords-in-region))
    :config
    (progn
      (spacemacs|hide-lighter sqlup-mode)
      (setq sqlup-blacklist (append sqlup-blacklist
                                    sql-capitalize-keywords-blacklist)))))
```

#### Reproduction guide :beetle:
- Start Emacs

*Observed behaviour:* :eyes: :broken_heart:

I got an error message:

```
Error (use-package): sqlup-mode/:init: Key sequence = c starts with non-prefix key =
```

*Expected behaviour:* :heart: :smile:
I want my emacs to start normally.

#### System Info :computer:
- OS: windows-nt
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. cf997e675)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: emacs
- Completion: ivy
- Layers:     
```elisp
(spacemacs-completion spacemacs-editing spacemacs-editing-visual spacemacs-modeline spacemacs-navigation spacemacs-org spacemacs-project spacemacs-visual auto-completion helm ivy better-defaults org lsp command-log graphviz yaml emacs-lisp markdown latex plantuml
                      (sql :variables sql-capitalize-keywords t)
                      (python :variables python-backend 'anaconda)
                      (c-c++ :variables c-c++-enable-google-style t c-c++-default-mode-for-headers 'c-mode)
                      my-csharp my-java my-web my-utils)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2

**Comment by Compro-Prasad** [comment]
Do we require both sqlfmt and sqlup?

**Comment by rgb-24bit** [comment]
This problem has been fixed by 072c01f
